project/jmh.scala

```diff
@@ -70,12 +42,18 @@ object RenaissanceJmh {
     nonGpl: Boolean,
     groupJars: Seq[(String, Seq[File], Seq[File])]
   ) = {
-    val perProjectBenchmarkClasses = for ((project, allJars, loadedJars) <- groupJars) yield {
+    val perProjectBenchmarkClasses = for {
+      (project, allJars, loadedJars) <- groupJars
+      // TODO: Filter projects in the build file if possible
+      if project.startsWith("benchmarks")
```

Comment:
Maybe you want to use `"benchmarks/"` as an extra precaution.

Reply:
Will do.